tests/clv/models/test_pareto_nbd.py

```diff
@@ -331,3 +334,97 @@ def test_expected_purchase_probability(self, test_n, test_t):
             est_purchases_new_customer.mean(("chain", "draw")),
             rtol=0.001,
         )
+
+    def test_dropout_purchase_distributions(self) -> None:
+        # TODO: Why isn't self.data working here?
```

Comment:
Unclear TODO
```suggestion
        # TODO: Why isn't self.data working here?
```

Reply:
See previous comment.

A `pytest` fixture for the test dataset could also be used for testing the `BetaGeoModel` and probably even resolve https://github.com/pymc-labs/pymc-marketing/issues/172, so it may be better to leave it for a future PR.